pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+testpaths = tests/
+addopts = -rsxX -vv
+log_format = %(asctime)s | %(levelname)s | %(name)s:%(lineno)d | %(message)s
+log_cli_level = ERROR
```

Comment:
Same as for DCC - accelerates `-k` test runs
https://github.com/plotly/dash-core-components/pull/740